### ISSUE TITLE
[Feature Request] Collection, Card tab and duplication cards

### ISSUE THREAD

**Issue by derornos** [zzorba/ArkhamCards#391]
1. Please change from whether i have that collection to how many collections i have.

If i have 4 core sets (3 core and 1 revised core set) and 2 dunwich sets (1 investigator set, 1 deluxe and 6 mythos packs), I cannot check the Collection correctly.

.

2. I would like to be able to know the quantity of the card based on the collection in the card tab.

When organizing physical cards, I sometimes want to know exactly how many cards there are. In this case, I believe the app will be able to help with this.

.

3. Is it possible to provide an option to show the duplication cards as well?

As you know, there are charisma cards in the revised core and Dunwich. It would be nice to have the option to view all of these cards by searching for them in the Card tab.

.

Thanks a lot.

**Comment by zzorba** [comment]
Yeah, I've held back on building this, mostly since I just have a ton of spots where I've special coded the collection logic.  

That said, at this point because of reprints, I always have to have the 'collection' info fetched to know quantity on a card.  I'll look into this.

As for #3, if you filter by pack (or click on Collection on settings and choose the pack), it should show the reprints for that pack.  Or do you mean, know which pack it was reprinted on when looking at a card?

**Comment by derornos** [comment]
I know you can see reprint cards per pack. However, it is not easy to recognize at a glance in the card tab.
By adding the On/Off function to the [...] button in the card tab, it would be nice if the user could choose whether to omit the reprint card or not.

For example, if I search for a 'chrisma' card in a state where reprint cards are also set to be displayed, I want it to be displayed in the card tab as follows.

![image](https://user-images.githubusercontent.com/15874357/184285735-8bd99f73-d5f2-4485-98ed-a4d155155d35.png)

**Comment by ombre5733** [comment]
This feature would be really useful for me as well.  I have the start investigators two times and do not like that the app warns about too many duplicates, when I actually have more than enough :-)
